.releaserc

```diff
@@ -1,9 +1,8 @@
 {
   "branches": [
-    "master",
-    "next",
-    { "name": "alpha", "prerelease": true },
-    { "name": "beta", "prerelease": true }
+    { "name": "release-22.x", "range": "22.x", "channel": "22.x" },
```

Comment:
Based on https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration, this should make it so
* `name`: we're looking at the `release-22.x` branch
* `range`: this only publishes releases in the `22.x` range of semantic versions
* `channel`: these releases are published to the `22.x` tag on `npm`